app/components/sidebars/main/channels_list/filtered_list/filtered_list.js

```diff
@@ -82,14 +86,13 @@ class FilteredList extends Component {
         return !deepEqual(this.props, nextProps, {strict: true}) || !deepEqual(this.state, nextState, {strict: true});
     }
 
-    componentWillReceiveProps(nextProps) {
-        if (this.props.term !== nextProps.term) {
-            const {actions, currentTeam} = this.props;
-            const {term} = nextProps;
+    static getDerivedStateFromProps(props, state) {
+        if (state.term !== props.term) {
+            const {actions, currentTeam} = state;
```

Comment:
not seeing why these are part of the state and according to the code they are `null`